src/ununpack/agent/Makefile.sa

```diff
@@ -0,0 +1,75 @@
+######################################################################
+# Copyright (C) 2020 Siemens AG
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# version 2 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+######################################################################
+
+TOP = ../../..
+VARS = $(TOP)/Makefile.conf
+DEPS = $(TOP)/Makefile.deps
+include $(VARS)
+LDFLAGS_LOCAL = -lmagic $(shell libgcrypt-config --libs)
+CFLAGS_LOCAL = -DSTANDALONE -Wall -D_FILE_OFFSET_BITS=64 $(shell libgcrypt-config --cflags) $(shell pkg-config --cflags --libs glib-2.0)
+EXE = departition-sa ununpack-sa
+
+CHKHDR = checksum.h
+CHKSRC = $(CHKHDR:%.h=%.c) traverse.c utils.c
+UUHDR = standalone.h ununpack.h ununpack-iso.h ununpack-disk.h ununpack-ar.h $(CHKHDR) ununpack_globals.h
+UUSRC = $(UUHDR:%.h=%.c)
+
+OBJECTS = standalone.o checksum.o traverse.o ununpack-iso.o ununpack-ar.o ununpack-disk.o utils.o
+COVERAGE = $(OBJECTS:%.o=%_cov.o)
+
+all: $(FOLIB) $(EXE)
+
+coverage: $(FOLIB) $(EXE:%=%-coverage)
+
+departition-sa: departition.c $(VARS)
+	$(CC) $< $(CFLAGS_LOCAL) $(FO_LDFLAGS) -o $@
+
+departition-coverage: departition.c $(VARS)
+	$(CC) $< $(CFLAGS_LOCAL) $(FO_LDFLAGS) $(FLAG_COV) -o $(@:%-coverage=%)
+
+ununpack-sa: ununpack.c libununpack.a $(VARS) $(DB) $(REPO) $(AGENTLIB) $(UUHDR)
+	$(CC) ununpack.c libununpack.a $(CFLAGS_LOCAL) $(LDFLAGS_LOCAL) $(DEFS) -o $@
+
+ununpack-coverage: ununpack.c libununpack_cov.a $(VARS) $(DB) $(REPO) $(AGENTLIB) $(UUHDR)
+	$(CC) ununpack.c libununpack_cov.a $(CFLAGS_LOCAL) $(LDFLAGS_LOCAL) $(FLAG_COV) $(DEFS) -o $(@:%-coverage=%)
+
+install: all $(VARS)
+	$(INSTALL_PROGRAM) departition-sa $(DESTDIR)$(MODDIR)/ununpack/agent/departition-sa
+	$(INSTALL_PROGRAM) ununpack-sa $(DESTDIR)$(MODDIR)/ununpack/agent/ununpack-sa
```

Comment:
Do you think installing stand-alone version along side normal version will be a nice idea?

Reply:
Now I see, we do it for nomos as well and it will be okay as normal `make` will not touch this file.